apps/api/src/bid/routes/bids/bids.router.ts

```diff
@@ -0,0 +1,34 @@
+import { createRoute } from "@hono/zod-openapi";
+import { container } from "tsyringe";
+
+import { BidController } from "@src/bid/controllers/bid/bid.controller";
+import { ListBidsParamsSchema, ListBidsResponseSchema } from "@src/bid/http-schemas/bid.schema";
+import { OpenApiHonoHandler } from "@src/core/services/open-api-hono-handler/open-api-hono-handler";
+
+const listRoute = createRoute({
+  method: "get",
+  path: "/v1/bids/{dseq}",
```

Comment:
`dseq` is not a unique identifier for a deployment. `userId` + `dseq` or `address/owner` + `dseq` is. In our case user seems to be more appropriate. So the interface should prob be something like:

```suggestion
  path: "/v1/bids/${userId}/{dseq}",
```

It's not perfect so it's totally open to discussion, however user should be taken into account

Reply:
yes, I understand this. However it's generally better to specify a user id anyway and then use permissions to limit the query or check if authorized. this endpoint then could be used according to permissions.

Reply:
yea, sounds good